test/e2e/apimachinery/resource_quota.go

```diff
@@ -1099,7 +1101,9 @@ var _ = SIGDescribe("ResourceQuota [Feature:PodPriority]", func() {
 	ginkgo.It("should verify ResourceQuota's priority class scope (quota set to pod count: 1) against a pod with same priority class.", func() {
 
 		_, err := f.ClientSet.SchedulingV1().PriorityClasses().Create(context.TODO(), &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: "pclass1"}, Value: int32(1000)}, metav1.CreateOptions{})
-		framework.ExpectEqual(err == nil || apierrors.IsAlreadyExists(err), true)
+		if err == nil && apierrors.IsAlreadyExists(err){
```

Comment:
This isn't the same logic. The following expressions are equivalent:

```
err == nil || apierrors.IsAlreadyExists(err) == true
!(err == nil || apierrors.IsAlreadyExists(err))
err != nil && !apierrors.IsAlreadyExists(err)
```

The last one is what we want (simplest).